src/BPSecLib_Public.h

```diff
@@ -118,6 +118,28 @@ typedef enum
     BSL_BUNDLECRCTYPE_32 = 2,
 } BSL_BundleCRCType_e;
 
+typedef enum
```

Comment:
Structures look good, can just add documentation, especially to explain what each index enum is meant to convey.